src/library/PiaFormat.ts

```diff
@@ -0,0 +1,94 @@
+import dayjs from "dayjs";
+
```

Comment:
Here, we should create a `PiaData` interface that specifies the data format we want to parse into:
```
enum PiaSex {
  Male = 0,
  Female = 1
}

interface PiaData {
  ssn: string
  sex: PiaSex
  birthDate: Date
}
```